CONTRIBUTING.rst

```diff
@@ -51,20 +49,12 @@ Requirements
 * pandas
 * seaborn
 
+For a complete list of requirements with information on the versions presently used
+to build the SciPy lecture notes, see ``requirements.txt``.
```

Comment:
Maybe a `:download:` link, so that whoever is looking at that on the web can download the requirements.txt file?

Reply:
Good point. An alternative would be to use a `literalinclude` of the requirements file. What do you think?